src/ode_solve.jl

```diff
@@ -29,7 +29,8 @@ of the physics-informed neural network which is used as a solver for a standard
 
 ## Example
 
-```julia
+```juliap = [1.5, 1.0, 3.0, 1.0]
```

Comment:
This looks like an error

Reply:
Not sure where that came from but its fixed now, sorry about that